### ISSUE TITLE
Suggested addition "void SetPlotRange(ImPlotRange)"

### ISSUE THREAD

**Issue by riclarsson** [epezent/implot#4]
Just spotted this library as I needed some plotting tools with imgui.  I like the work.  I made the following changes to allow dynamic resizing of windows and I wonder.

For implot.h-file
```cpp
/// Set the axes limits of this plot
void SetPlotLimits(ImPlotLimits limits, int y_axis=-1);
```

For implot.cpp-file
```cpp
void SetPlotLimits(ImPlotLimits limits, int y_axis_in) {
    IM_ASSERT_USER_ERROR(y_axis_in >= -1 && y_axis_in < MAX_Y_AXES, "y_axis needs to between -1 and MAX_Y_AXES");
    IM_ASSERT_USER_ERROR(gp.CurrentPlot != NULL, "GetPlotLimits() Needs to be called between BeginPlot() and EndPlot()!");
    const int y_axis = y_axis_in >= 0 ? y_axis_in : gp.CurrentPlot->CurrentYAxis;

    ImPlot& plot = *gp.CurrentPlot;
    plot.XAxis.Range = limits.X;
    plot.YAxis[y_axis].Range = limits.Y;
}
```

I am new to imgui so I do not know how to include the `ImGuiCond` statements here.

Use case: resizing by numbers from a text-box or slider.  This requires seeing the plot before knowing what to do with the ImPlotRange.  Example code added after an EndPlot() call to demonstrate usefulness:

```cpp
ImPlotLimits range = ImGui::GetPlotLimits();
bool change=false;
change |= ImGui::InputFloat("X min", &range.X.Min, 0, 0, "%g"); 
change |= ImGui::InputFloat("X max", &range.X.Max, 0, 0, "%g");
change |= ImGui::InputFloat("Y min", &range.Y.Min, 0, 0, "%g");
change |= ImGui::InputFloat("Y max", &range.Y.Max, 0, 0, "%g");
if (change) ImGui::SetPlotLimits(range);
```

edit: update code to version from yesterday (warning, no testing of last function!)

**Comment by epezent** [comment]
I see your issue, and will add something along these lines soon. In case you weren't aware, you can open a context menu by double right clicking the plot. This lets you modify axis ranges:

![image](https://user-images.githubusercontent.com/29577475/80503741-ec5dd380-8937-11ea-914d-aaa3fd91d49b.png)

**Comment by riclarsson** [comment]
Thank you for the right-click feature.  I did not know about it and it is really useful.  It replaces the need of the menu in my comment above (I updated the code to reflect changes I have seen in this library up until yesterday).  I will see if I can add a copy-paste to the right-click dropdown menu.  My use case is very special but I want to combine this with a sensor taking physical measurements, and it helps if I can copy an axis from one window+plot to another window+plot.
